pkg/resources/stream_on_table_acceptance_test.go

```diff
@@ -595,7 +593,6 @@ func TestAcc_StreamOnTable_InvalidConfiguration(t *testing.T) {
 		TerraformVersionChecks: []tfversion.TerraformVersionCheck{
 			tfversion.RequireAbove(tfversion.Version1_5_0),
 		},
```

Comment:
Shouldn't it be changed to StreamOnTable checkDesctroy instead of removed completely?

Reply:
This test case doesn't create any resources because of invalid configuration, so there's no need to use checkDestroy.